webapp/app/forms/steps/lotse/personal_data.py

```diff
@@ -124,6 +124,20 @@ def validate_request_new_tax_number(form, field):
             else:
                 validators.Optional()(form, field)
 
+        def validate(self, extra_validators=None):
+            all_fields_are_valid = super().validate(extra_validators)
+
+            if not all_fields_are_valid and self.steuernummer.errors:
+                return all_fields_are_valid  # Only validate tax number if all other validations of tax number are correct
```

Comment:
>Are you sure you want to retur "all_fields_are_valid"? or maybe "return False"?

The idea was the following:
If there are any errors on a field for the Steuernummer, we should not check the tax number with erica because the user should first fix those problems (e.g. the field is empty). In these cases we will just return the result of the normal validation. However, we can also make it more explicit and return False, because this will always be False anyway.

> And don't you want to check if all_fields_are_valid is false OR self.steuernummer.errors exist?
That is a good question. The more I think about it, we could also just check for `self.steuernummer.errors exist` because we should always receive all_fields_are_valid == False in that case. That said, I do not want to return a valid result without either being sure that there is a problem with the Steuernummer or being sure that I have checked it's validity.

Besides all that: Do you think that we should rename `all_fields_are_valid`?